server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/GetMessages.feature

```diff
@@ -199,3 +199,11 @@ Feature: GetMessages method
     And the preview of the message is "multipart/related content"
     And the property "textBody" of the message is null
     And the htmlBody of the message is "<html>multipart/related content</html>\n"
+
+  Scenario: Retrieving message should return textBody and htmlBody when incoming mail have an inlined HTML and text body without Content-ID
+    Given the user has a message in "inbox" mailbox from file "eml/inlinedMultipart.eml"
```

Comment:
It's not ok to do that, the description of the test is no longer self-describing. Please add a way to describe the structure of the email you want to generate.